packages/bot/src/utils/music/autoplay/recommendationSourceMapping.ts

```
@@ -0,0 +1,37 @@
+		case 'lastfm-loved':
+			return PrismaRecommendationSource.LASTFM_LOVED
+		case 'lastfm-similar':
+			return PrismaRecommendationSource.LASTFM_SIMILAR
+		case 'lastfm-genre-fallback':
+			return PrismaRecommendationSource.LASTFM_GENRE_FALLBACK
+		case 'artist-fallback':
+			return PrismaRecommendationSource.ARTIST_FALLBACK
+		case 'genre-tag':
+			return PrismaRecommendationSource.GENRE_TAG
+		default:
+			const _exhaustive: never = src
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)